maven/README.txt

```diff
@@ -1,9 +1,9 @@
 push-gwt.sh packages and deploys GWT to a local or remote Maven
 repository
  
-To build GWT from trunk and deploy to your local repo:
+To build GWT from main and deploy to your local repo:
 
-> cd trunk
+> git checkout main
 > ant clean dist-dev
 > maven/push-gwt.sh
```

Comment:
One more request (and this is also my fault for changing it), can you adjust the push-gwt script name to be `maven/push-gwtproject.sh`? The old push-gwt will only push com.google.gwt artifacts, which we don't expect to release again. I failed to update this correctly when I made that change for the last release.

Reply:
no problem